moveit_setup_assistant/CMakeLists.txt

```diff
@@ -1,169 +1,200 @@
-  src/tools/collision_linear_model.cpp
-  src/tools/collision_matrix_model.cpp
-  src/tools/rotated_header_view.cpp
+#  src/tools/collision_linear_model.cpp
+#  src/tools/collision_matrix_model.cpp
+#  src/tools/rotated_header_view.cpp
 )
-set_target_properties(${PROJECT_NAME}_tools PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${PROJECT_NAME}_tools
-  ${YAMLCPP_LIBRARIES}
-  ${catkin_LIBRARIES}
-  ${Boost_LIBRARIES}
-  ${QT_LIBRARIES}
+target_include_directories(${PROJECT_NAME}_tools PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>
+  ${OMPL_INCLUDE_DIRS}
+)
+ament_target_dependencies(
+  ${PROJECT_NAME}_tools
+  "moveit_core"
+  "moveit_ros_planning"
+  "moveit_ros_visualization"
+  "rviz2"
+  "rviz_common"
+  "rclcpp"
+  "urdf"
+  "srdfdom"
 )
 
 # Main Widgets Library - all screens (navigation options)
 add_library(${PROJECT_NAME}_widgets
   src/widgets/start_screen_widget.cpp
-  src/widgets/planning_groups_widget.cpp
-  src/widgets/double_list_widget.cpp
-  src/widgets/kinematic_chain_widget.cpp
-  src/widgets/group_edit_widget.cpp
-  src/widgets/default_collisions_widget.cpp
-  src/widgets/robot_poses_widget.cpp
-  src/widgets/end_effectors_widget.cpp
-  src/widgets/virtual_joints_widget.cpp
-  src/widgets/passive_joints_widget.cpp
-  src/widgets/perception_widget.cpp
-  src/widgets/configuration_files_widget.cpp
-  src/widgets/ros_controllers_widget.cpp
-  src/widgets/controller_edit_widget.cpp
+#  src/widgets/planning_groups_widget.cpp
+#  src/widgets/double_list_widget.cpp
+#  src/widgets/kinematic_chain_widget.cpp
+#  src/widgets/group_edit_widget.cpp
+#  src/widgets/default_collisions_widget.cpp
+#  src/widgets/robot_poses_widget.cpp
+#  src/widgets/end_effectors_widget.cpp
+#  src/widgets/virtual_joints_widget.cpp
+#  src/widgets/passive_joints_widget.cpp
+#  src/widgets/perception_widget.cpp
+#  src/widgets/configuration_files_widget.cpp
+#  src/widgets/ros_controllers_widget.cpp
+#  src/widgets/controller_edit_widget.cpp
   src/widgets/navigation_widget.cpp
   src/widgets/header_widget.cpp
   src/widgets/setup_assistant_widget.cpp
   src/widgets/setup_screen_widget.cpp
-  src/widgets/simulation_widget.cpp
-  src/widgets/author_information_widget.cpp
   ${HEADERS}
 )
-set_target_properties(${PROJECT_NAME}_widgets PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 target_link_libraries(${PROJECT_NAME}_widgets
-  ${PROJECT_NAME}_tools
-  ${QT_LIBRARIES}
-  ${catkin_LIBRARIES}
-  ${Boost_LIBRARIES}
+    ${PROJECT_NAME}_tools
+)
+target_include_directories(${PROJECT_NAME}_widgets PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+set_target_properties(${PROJECT_NAME}_widgets PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+ament_target_dependencies(
+  ${PROJECT_NAME}_widgets
+  "moveit_core"
+  "moveit_ros_planning"
+  "moveit_ros_visualization"
+  "rviz2"
+  "rviz_common"
+  "rclcpp"
+  "urdf"
+  "srdfdom"
 )
```

Comment:
Please just leave this section as a commented-out version of the old cmake file here as this is not a port of the QT widgets.

Also when I try to build this project locally on Rolling I get this error:
```
In file included from /home/tyler/code/m2/ws_acm/install/moveit_ros_visualization/include/moveit/rviz_plugin_render_tools/robot_state_visualization.h:41,
                 from /home/tyler/code/m2/ws_acm/install/moveit_ros_visualization/include/moveit/robot_state_rviz_plugin/robot_state_display.h:43,
                 from /home/tyler/code/m2/ws_acm/src/moveit2/moveit_setup_assistant/src/widgets/setup_assistant_widget.cpp:57:
/home/tyler/code/m2/ws_acm/install/moveit_ros_visualization/include/moveit/rviz_plugin_render_tools/octomap_render.h:45:10: fatal error: OGRE/OgrePrerequisites.h: No such file or directory
   45 | #include <OGRE/OgrePrerequisites.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
```